agent/Dockerfile

```diff
@@ -24,11 +24,12 @@ RUN apt-get update && apt-get upgrade -y && \
 
 ADD ./ /opt/src/github.com/google/inverting-proxy
 
-RUN wget -O /opt/go1.14.2.linux-amd64.tar.gz \
-      https://storage.googleapis.com/golang/go1.14.2.linux-amd64.tar.gz && \
-    tar -C /usr/local -xzf /opt/go1.14.2.linux-amd64.tar.gz && \
+RUN wget -O /opt/go1.18.1.linux-amd64.tar.gz \
+      https://storage.googleapis.com/golang/go1.18.1.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf /opt/go1.18.1.linux-amd64.tar.gz && \
     export PATH=${PATH}:/usr/local/go/bin/:/opt/bin/ && \
     export GOPATH=/opt/ && \
+    go env -w GO111MODULE=off && \
```

Comment:
This is surprising. We have a go.mod file. If that one isn't working we should fix it.

Reply:
Fixed the go.mod